torchvision/prototype/transforms/utils.py

```diff
@@ -50,18 +56,25 @@ def query_spatial_size(flat_inputs: List[Any]) -> Tuple[int, int]:
     if not sizes:
         raise TypeError("No image, video, mask or bounding box was found in the sample")
     elif len(sizes) > 1:
+        # (This comment applies to the other query_() utils):
+        # Does this happen?
+        # What can go terribly wrong if we don't raise an error?
+        # Should we just document that this returns the size of the first inpt to avoid an error?
         raise ValueError(f"Found multiple HxW dimensions in the sample: {sequence_to_str(sorted(sizes))}")
     h, w = sizes.pop()
     return h, w
 
 
+# when can types_or_checks be a callable?
 def check_type(obj: Any, types_or_checks: Tuple[Union[Type, Callable[[Any], bool]], ...]) -> bool:
     for type_or_check in types_or_checks:
+        # That messed with my brain for a bit - add parenthesis?
         if isinstance(obj, type_or_check) if isinstance(type_or_check, type) else type_or_check(obj):
             return True
     return False
 
 
+# Are these public because they are needed for users to implement custom transforms?
```

Comment:
They are not strictly needed, but make the input checking a lot easier. If users come and look at our source how we do it, we should give them the same tools.